source/help/getting-started/creating-an-account.rst

```diff
@@ -0,0 +1,53 @@
+=================================================
+Creating an account and signing in the first time
+=================================================
+
+It's a relatively straight forward process to create an account on Mattermost and start communicating with other people. You enter some basic information on the account creation page then click **Create Account**.
+
+.. Important::
+  - The account creation process might be different than described here, depending on the policies of your organization.
+  - If your administrator sent you instructions, use those instructions.
+  - If you are not able to create an account using these instructions, contact your administrator.
+
+On the account creation page, you must enter the following information:
+
+email address
```

Comment:
I'd like to reserve title case for page titles. How about
Email address
Username
Password